polygeist/tools/cgeist/Test/Verification/sycl/single_task.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: sycl-clang.py %s -S -emit-llvm -o %t.ll
+// Test that the LLVMIR generated is verifiable.
+// RUN: opt -verify -disable-output < %t.ll
+// RUN: cat %t.ll | FileCheck %s
```

Comment:
%t.ll should be removed after the last RUN step. Currently is left over.

Reply:
I think it is common for %t to left over.

Reply:
I do not feel strongly about this but I would expect the directory containing the generated scripts to be unchanged after running the test.

Reply:
If it is %s, I agree we should not generate temp files in the test case directory, and if we do, we should have it removed.
I found it convenient to have the generated temp files in the build directory, and it is a common behavior for %t, so keeping it.